main.js

```diff
@@ -1,4 +1,137 @@
 /* eslint-disable */
+const filterBtn = document.getElementById('filterBtn-js');
+const filter = document.getElementById('filter-js');
+
+filterBtn.addEventListener('click', () => {
+  filter.style.display = 'block';
+})
+
+const adultCount = document.getElementById('adultCount');
+const childrenCount =document.getElementById('childrenCount');
+const roomCount = document.getElementById('roomCount');
+const minusBtn = document.querySelectorAll('.minus-filter-btn');
+const plusBtn = document.querySelectorAll('.plus-filter-btn');
+const childAgeBlock = document.querySelectorAll('.filter-childAgeBlock');
+
+//Adults
+
+minusBtn[0].addEventListener('click', () => {
+  if (adultCount.textContent !== `${0}`) {
+    adultCount.textContent = `${adultCount.textContent - 1}`;
+  }
+  if (adultCount.textContent === `${0}`) {
+    minusBtn[0].style.border = '1px solid #CECECE';
+    minusBtn[0].style.color = '#CECECE';
+  }
+  if(adultCount.textContent !== `${30}`) {
+    plusBtn[0].style.border = '1px solid #3077C6';
+    plusBtn[0].style.color = '#3077C6';
+  }
+  filterBtn.textContent = `${adultCount.textContent} Adults — ${childrenCount.textContent} Children — ${roomCount.textContent} Room`;
+})
+
+plusBtn[0].addEventListener('click', () => {
+  if(adultCount.textContent !== `${30}`) {
+    adultCount.textContent = `${Number(adultCount.textContent) + 1}`;
+  }
+  if (adultCount.textContent !== `${0}`) {
+    minusBtn[0].style.border = '1px solid #3077C6';
+    minusBtn[0].style.color = '#3077C6';
+  }
+  if (adultCount.textContent === `${30}`) {
+    plusBtn[0].style.border = '1px solid #CECECE';
+    plusBtn[0].style.color = `#CECECE`;
+  }
+  filterBtn.textContent = `${adultCount.textContent} Adults — ${childrenCount.textContent} Children — ${roomCount.textContent} Room`;
+})
+
+//Children
+if (childrenCount.textContent === `${0}`) {
+  minusBtn[1].style.border = '1px solid #CECECE';
+  minusBtn[1].style.color = '#CECECE';
+}
```

Comment:
If you will be using styles as a classes you can add default styles in html file for +/- buttons and this code can be removed